controllers/health.go

```diff
@@ -0,0 +1,17 @@
+package controllers
+
+import (
+	"encoding/json"
+	"gopkg.in/macaron.v1"
+	"net/http"
+	"os"
+)
+
+func GetHealth(ctx *macaron.Context) {
```

Comment:
I'd prefer if this was actually named `GetStatusz`, if that's OK? It should be under `/statusz`